src/backend/gporca/libgpopt/src/base/CDistributionSpecReplicated.cpp

```diff
@@ -13,18 +13,23 @@ using namespace gpopt;
 //	Check if this distribution spec satisfies the given one
 //	based on following satisfiability rules:
 //
-//	pprs = requested distribution
+//	pds = requested distribution
 //	this = derived distribution
 //
 //	Table - Distribution satisfiability matrix:
 //	  T - Satisfied
 //	  F - Not satisfied; enforcer required
+//
+//	  Note that when checking for satisfiability for TaintedReplicated,
+//	  we don't call the Match() function, since TaintedReplicated is a
+//	  derived only property. Hence if FSatisfies is being called on
+//	  TaintedReplicated, pds can never be TaintedReplicated.
 //	+-------------------------+------------------+-------------------+
-//	|                         | Replicated       | TaintedReplicated |
+//	|                         | StrictReplicated | TaintedReplicated |
 //	+-------------------------+------------------+-------------------+
 //	| Matches                 | T                | (default) F       |
-//	| NonSingleton            | FAllowReplicated | same              |
-//	| GeneralReplicated       | T                | T                 |
+//	| NonSingleton            | FAllowReplicated | FAllowReplicated  |
+//	| Replicated              | T                | T                 |
```

Comment:
I think there is another line missing for a "strict replicated" requirement:
```suggestion
//	| Replicated              | T                | T                 |
//	| Strict Replicated       | T                | F                 |
```

Reply:
Done.